--- ./Makefile.org	Thu Jul 10 20:42:12 2003
+++ ./Makefile	Mon Oct 13 18:05:37 2003
@@ -2,12 +2,12 @@
 #
 # Simple makefile for chpwdfile project
 
-CC=gcc
-#COPTS=-Wall -fpedantic
-COPTS=-s -O2
+CC	= gcc
+#COPTS	= -Wall -fpedantic
+COPTS	= -s -O2
 
-BINDIR=/usr/local/bin
-MANDIR=/usr/local/man/man1
+BINDIR	= /usr/local/bin
+MANDIR	= /usr/local/man/man1
 
 chpwdfile: chpwdfile.c util.c chpwdfile.h util.h
 	$(CC) $(COPTS) -lcrypt -o chpwdfile chpwdfile.c util.c
@@ -23,16 +23,16 @@
 	-rm chpwdfile chpwdfile-cgi
 
 install:
-	if [ ! -d $(BINDIR) ] ; then mkdir -p $(BINDIR) ; fi
-	cp chpwdfile $(BINDIR)
-	ln -sf $(BINDIR)/chpwdfile $(BINDIR)/pwadd
-	ln -sf $(BINDIR)/chpwdfile $(BINDIR)/pwdel
-	ln -sf $(BINDIR)/chpwdfile $(BINDIR)/pwmod
-	if [ ! -d $(MANDIR) ] ; then mkdir -p $(MANDIR) ; fi
-	cp chpwdfile.1 $(MANDIR)
-	ln -sf $(MANDIR)/chpwdfile.1 $(MANDIR)/pwadd.1
-	ln -sf $(MANDIR)/chpwdfile.1 $(MANDIR)/pwdel.1
-	ln -sf $(MANDIR)/chpwdfile.1 $(MANDIR)/pwmod.1
+	if [ ! -d $(DESTDIR)$(BINDIR) ] ; then mkdir -p $(DESTDIR)$(BINDIR) ; fi
+	cp chpwdfile $(DESTDIR)$(BINDIR)
+	ln -sf chpwdfile $(DESTDIR)$(BINDIR)/pwadd
+	ln -sf chpwdfile $(DESTDIR)$(BINDIR)/pwdel
+	ln -sf chpwdfile $(DESTDIR)$(BINDIR)/pwmod
+	if [ ! -d $(DESTDIR)$(MANDIR) ] ; then mkdir -p $(DESTDIR)$(MANDIR) ; fi
+	cp chpwdfile.1 $(DESTDIR)$(MANDIR)
+	echo '.so chpwdfile.1' > $(DESTDIR)$(MANDIR)/pwadd.1
+	echo '.so chpwdfile.1' > $(DESTDIR)$(MANDIR)/pwdel.1
+	echo '.so chpwdfile.1' > $(DESTDIR)$(MANDIR)/pwmod.1
 
 # PRCS is a fine thing
 # $Format: "DISTDIR = chpwdfile-$ProjectVersion$"$
